prac_09/taxi.py

```diff
@@ -0,0 +1,34 @@
+"""
+CP1404/CP5632 Practical
+Car class
+"""
+from prac_09.car import Car
+
+
+class Taxi(Car):
+    """Specialised version of a Car that includes fare costs."""
+    price_per_km = 1.23  # Task 1
+
+    def __init__(self, name, fuel):  # Task 2
+        """Initialise a Taxi instance, based on parent class Car."""
+        super().__init__(name, fuel)
+        self.current_fare_distance = 0
+
+    def __str__(self):
+        """Return a string like a Car but with current fare distance."""
+        return f"{super().__str__()}, {self.current_fare_distance}km on current fare, ${self.price_per_km:.2f}/km"
+
+    def get_fare(self):
+        """Return the price for the taxi trip, rounded to the nearest 10."""
+        fare = super().get_fare()  # Updated to round fare to the nearest 10.
```

Comment:
Is there a get_fare method in the parent class? I'm confused on what this will do